app/clients/zendesk_sell_client.py

```diff
@@ -0,0 +1,94 @@
+import json
+import requests
+
+from typing import Dict, List, Union
+
+from flask import current_app
+
+from app.authentication.bearer_auth import BearerAuth
+from app.user.contact_request import ContactRequest
+
+__all__ = [
+    'ZenDeskSellClient'
+]
+
+
+class ZenDeskSellClient(object):
```

Comment:
Does it need to be suffixed with `Client`? Can follow the same naming convention as other classes in `app/clients`